apps/playground/src/app/components/forms/file-attachment/file-attachment.component.html

```diff
@@ -16,3 +16,15 @@
     </sky-file-item>
   </div>
 </div>
+
+<form [formGroup]="formGroup">
+  <sky-file-attachment
+    required="true"
+    labelText="Single file attachment"
+    hintText="Sample hint text"
+    [acceptedTypes]="acceptedTypes"
+    [maxFileSize]="maxFileSize"
+    [validateFn]="validateFile"
+    (fileChange)="onFileChange($event)"
+  ></sky-file-attachment>
```

Comment:
Can be self-closing. Also, can you check the alphabetical order of the attributes?

Reply:
updated